dotnet/samples/KernelSyntaxExamples/Getting_Started/Step2_Add_Plugins.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+// ReSharper disable once InconsistentNaming
+using System;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+/**
+* This example shows how to load a <see cref="KernelPlugin"/> instances.
+*/
+// ReSharper disable once InconsistentNaming
+public static class Step2_Add_Plugins
+{
+    /// <summary>
+    /// Shows different ways to load a <see cref="KernelPlugin"/> instances.
+    /// </summary>
+    public static async Task RunAsync()
+    {
+        // Create a kernel with OpenAI chat completion
+        IKernelBuilder kernelBuilder = Kernel.CreateBuilder();
+        kernelBuilder.AddOpenAIChatCompletion(
+                modelId: TestConfiguration.OpenAI.ChatModelId,
+                apiKey: TestConfiguration.OpenAI.ApiKey);
+        kernelBuilder.Plugins.AddFromType<TimeInformation>();
```

Comment:
If we had `AddPlugins` we wound't have to specify a `kernelBuilder` variable, would be way simpler.

Reply:
Agreed, we can add this later